### PR TITLE
Fix disable attribute of fieldConfig

### PR DESCRIPTION
**Pull request by SHA-4**
Added support for the disabled attribute for all types

### CODE REVIEW

**Comment by rkuykendall** [comment]
So reading this, I thought "wait, doesn't fieldConfig have a disabled attribute?" and yes, yes it does. But it's... not implemented? It looks like it's a holdover from V2 where it IS implemented. I think you could add this line, change the test from `editProps: { disabled: true }` to `disabled: true` and be done?

image

OR, we should remove it from fieldConfig. But it seems important and universal enough to be included top-level. Agree we should implement it for everything else soon.

File: src/building-blocks/FormField.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ class FormField extends Component<IFormFieldProps> {
       ;
 
     return {
+      disabled: fieldConfig.disabled,
```
Comment (rpalermodrums) [inline src/building-blocks/FormField.tsx]: Why is this necessary? won't this be in `editProps`?

File: src/inputs/Address.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ class Address extends Component<IAddressProps> {
   }
 
   private get fieldSet () {
-    const { fieldConfig: { colProps, required, stateProps } } = this.injected
+    const { fieldConfig: { colProps, required, stateProps, disabled } } = this.injected
```
Comment (rpalermodrums) [inline src/inputs/Address.tsx]: I thought that `Address` was outside of this use case?

File: src/building-blocks/FormField.tsx (hunk 1 above)
Comment (rkuykendall) [reply src/building-blocks/FormField.tsx]: Na, it's a core attribute:
image

File: src/inputs/Address.tsx (hunk 2 above)
Comment (rkuykendall) [reply src/inputs/Address.tsx]: It was easy to implement it for all types.
